src/sinks/greptimedb/service.rs

```diff
@@ -0,0 +1,153 @@
+use std::num::NonZeroUsize;
+use std::sync::Arc;
+use std::task::Poll;
+
+use futures_util::future::BoxFuture;
+
+use greptimedb_client::api::v1::auth_header::AuthScheme;
+use greptimedb_client::api::v1::*;
+use greptimedb_client::{Client, Database, Error as GreptimeError};
+use tower::Service;
+use vector_common::finalization::{EventFinalizers, EventStatus, Finalizable};
+use vector_common::internal_event::CountByteSize;
+use vector_common::request_metadata::{MetaDescriptive, RequestMetadata};
+use vector_core::event::Metric;
+use vector_core::stream::DriverResponse;
+
+use crate::sinks::prelude::RetryLogic;
```

Comment:
Can you change it to:

```suggestion
use crate::sinks::prelude::*;
```

Then there are a whole bunch of other imports that can be removed. This should be done in the other sink files too.